src/content/docs/tutorial-errors/respond-outages.mdx

```diff
@@ -0,0 +1,155 @@
+
+import apiGateway from 'images/apm_screenshot-crop_api-gateway-summary.webp'
+
+import apmErrorsInboxPage from 'images/apm_screenshot-crop_errors-inbox-page.webp'
+
+import apmErrorsAnomaly from 'images/apm_screenshot-crop_errors-anomaly.webp'
+
+Errors are bound to happen. Even with an observability tool, finding the source of an error isn't as straightforward as you might assume. Think about a flooded yard: you notice water flowing near your hose bib, but the cause of the flood is actually a crack somewhere in your water main. If you assumed that the leaking hose caused the flood, you'd end up with a fixed hose but a ruined lawn. The mistake would be costly. 
+
+Error analysis takes you to the source of the issue so you can fix the crack before your system is flooded. When a team makes a new deployment or a service fails upstream, you need to dig deeper before implementing any solutions. There's no room for assumptions in error analysis.
+
+## Objectives [#objective]
+
+This tutorial series shows you how to solve critical errors, then guides you into reducing your overall error count. This doc covers these concepts:
+
+* Choosing a service to begin error analysis
+* Choosing an error group that indicates an outage 
+
+## Prerequisites [#prereq]
+
+To monitor your application's performance, you will use an agent created specifically for your app's language. Click a logo to install an agent. This will send you to the New Relic platform where you will be guided through installing and configuring the agent.
+
+<TechTileGrid>
+  
+  <TechTile
+    name="Go agent"
+    icon="logo-go"
+    to="https://one.newrelic.com/nr1-core?state=985d4005-ba90-a8c7-1da1-2af34539b03b"
+  />
+
+  <TechTile
+    name="Java agent"
+    icon="logo-java"
+    to="https://one.newrelic.com/nr1-core?state=80d18bcb-4919-1fcb-2b77-9406838eb916"
+  />
+
+  <TechTile
+    name=".NET agent"
+    icon="logo-dotnet"
+    to="https://one.newrelic.com/nr1-core?state=30e93090-6dfa-6b70-8e75-472f54414355"
+  />
+
+  <TechTile
+    name="Node.js agent"
+    icon="logo-nodejs"
+    to="https://one.newrelic.com/marketplace/install-data-source?state=be2e62fa-cc3b-c428-27c4-8d662c9e80a1"
+  />
+
+  <TechTile
+    name="PHP agent"
+    icon="logo-php"
+    to="https://one.newrelic.com/nr1-core?state=aa633b41-72d4-009c-3abf-55dcf64894fe"
+  />
+
+  <TechTile
+    name="Python agent"
+    icon="logo-python"
+    to="https://one.newrelic.com/nr1-core?state=20fda75b-58fb-a92a-f9e1-7b052035c6e8"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+
+</TechTileGrid>
+
+Once you've installed an agent, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core?filters=(domain%3D'APM'ANDtype%3D'APPLICATION'))** and select your app. If you don't see much data just yet, step away for a while and let the agent gather real-time data as your application runs.
```

Comment:
how about "if you don't see much data, give the agent some additional time to gather data..."?